buildlib/azure-pipelines-release-drp.yml

```diff
@@ -6,9 +6,17 @@ pr:
   - master
   - v*.*.x
 
+schedules:
+- cron: '0 0 * * 6'
+  displayName: Saturday night test
+  always: true
+  branches:
+    include:
+    - master
+
 variables:
   DOCKER_OPT_VOLUMES: -v /hpc/local:/hpc/local
-  REPO_MIRROR: harbor-pdc.nvidia.com
+  REPO_MIRROR: harbor.mellanox.com
```

Comment:
Lets wait for the PDC harbor to work (next week) before we merge it

Reply:
Updated